### ISSUE TITLE
Dom5-style flags

### ISSUE THREAD

**Issue by Ealb** [elmokki/nationgen#897]
At some point, the higher-res Dom5-style flags should become the norm. This is low priority as far as I'm concerned, but eventually it should happen.

**Comment by elmokki** [comment]
I think these would without code changes or at least almost without them if all the flag files were just replaced with higher res ones.

There's some work in getting the 3dfying effect layer to look correct but that's nothing too horrible. Mostly it's just awful work of having to make flags and patterns and such.

**Comment by Ealb** [comment]
Yeah, it doesn't look hard - just annoying.

**Comment by elmokki** [comment]
So, Dom5 dump contained already 3dfied flags and thus things could be simplified. I did very little actual graphical work with the flags so now they only vary in color and the top ornament, not all of which I bothered to define. Ideally there'd be more generic top ornaments, but I think I might have to make some flag patterns too since those seem to be a good way to vary flags without making them look too much like vanilla flags. The icons in the middle of the flag I never really liked and with the flags already 3dfied adding them would be more annoying too, not that the effect showed too well before either.
